effect.lua

```diff
@@ -0,0 +1,49 @@
+function HandleEffectCommand(Split, Player)
+	local EffectID = tonumber(Split[3])
+	local Amplifier = tonumber(Split[5])
+	local ApplyEffect = function(OtherPlayer)
+		local Seconds = tonumber(Split[4]) * 20
+		OtherPlayer:AddEntityEffect(EffectID, Seconds, Amplifier)
+		if Split[2] ~= "@a" then
+			Player:SendMessageSuccess("Successfully added effect to player \"" .. OtherPlayer:GetName() .. "\"")
+		end
+	end
+	local RemoveEffects = function(OtherPlayer)
+		OtherPlayer:ClearEntityEffects()
+		if Split[2] ~= "@a" then
+			Player:SendMessageSuccess("Successfully removed effects from player \"" .. OtherPlayer:GetName() .. "\"")
+		end
+	end
+	if Split[3] ~= "clear" then
+		if Split[5] == nil then
+			Player:SendMessageInfo("Usage: " .. Split[1] .. " <player> <effect> [seconds] [amplifier] OR " .. Split[1] .. " <player> clear")
```

Comment:
Arguments in square brackets are optional? If yes then you have a wrong check above the usage `if Split[5] == nil then`